client/src/modules/dashboard/Events/pages/EventsPage.tsx

```diff
@@ -12,8 +21,30 @@ import { useAuth } from '../../../auth/store';
 import { useDashboardEventsQuery } from '../../../../generated/graphql';
 import { NextPageWithLayout } from '../../../../pages/_app';
 
+const ShowCanceledSwitch = ({
+  setShowCanceled,
+  defaultChecked,
+}: {
+  setShowCanceled: React.Dispatch<React.SetStateAction<boolean>>;
+  defaultChecked: boolean;
+}) => {
+  return (
+    <Flex>
+      <FormLabel htmlFor="show-canceled-events">Show canceled events</FormLabel>
```

Comment:
```suggestion
      <FormLabel htmlFor="show-canceled-events" marginBottom="0">Show canceled events</FormLabel>
```

Reply:
I'm not sure about this change, the `0` makes it a bit squeezed.
Before:
![image](https://user-images.githubusercontent.com/60067306/210581936-dd76aa0d-3e24-4924-8eda-f303370e4b4b.png)
After:
![image](https://user-images.githubusercontent.com/60067306/210581958-5f92e511-6df8-4250-a14d-d352c5fc77e3.png)

Reply:
the squeeze to zero is for the alignment in layout bigger than `base`, we can go with `marginBottom={{base: "", sm: "0"}}`